src/Functional.CQS.AOP.IoC.PureDI.Caching/Extensions/ExtensionsFor_IAsyncQueryHandler.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Functional.CQS.AOP.Caching.Infrastructure;
+
+namespace Functional.CQS.AOP.IoC.PureDI.Caching.Extensions
+{
+	/// <summary>
+	/// Extension methods for <see cref="IAsyncQueryHandler{TQuery,TResult}"/>.
+	/// </summary>
+	// ReSharper disable once InconsistentNaming
+	internal static class ExtensionsFor_IAsyncQueryHandler
```

Comment:
Why the inconsistent naming for the extension class?

Reply:
I'll make things consistent with other extension method classes across the repo.

ReSharper complains about `IAsyncQueryHandlerExtensions` (because it's not an interface) and `ExtensionsFor_IAsyncQueryHandler` (because of `_`).  Some time ago, I had opted for `ExtensionsFor_IInterface` when writing extension methods for interfaces.  Apparently I bounce back and forth between the two conventions.